### PR TITLE
feat: add `PageProps` to _app

### PR DESCRIPTION
**Pull request by huai-jie**
This should same purpose as https://github.com/denoland/fresh/pull/1153 but with more strict `type`

> Could use it as reference for  https://github.com/denoland/fresh/pull/1153 and close this as duplicate

### CODE REVIEW

File: src/server/types.ts [hunk 1]
```diff
@@ -116,7 +116,7 @@ export interface Route<Data = any> {
 
 // --- APP ---
 
-export interface AppProps {
+export interface AppProps extends PageProps {
```
Comment (lucacasonato) [inline src/server/types.ts:119]: `data` is always `unknown` I guess?

**Review by lucacasonato — APPROVED** [review]
LGTM

File: src/server/types.ts (hunk 1 above)
Comment (huai-jie) [reply src/server/types.ts:119]: Yup, `type` parameter is fine as well for `data` property

**Review by iuioiua — APPROVED** [review]
LGTM! Thanks, @huai-jie.

File: src/server/types.ts (hunk 1 above)
Comment (J053Fabi0) [reply src/server/types.ts:119]: Is not `unknown`, is `any`.

**Pull request merged**